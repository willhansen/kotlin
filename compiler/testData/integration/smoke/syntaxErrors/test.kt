package test

fun test() {
    konst s = System.in
}