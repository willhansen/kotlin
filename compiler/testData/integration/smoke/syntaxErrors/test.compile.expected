
ERR:
test.kt:4:20: error: expecting an element
    konst s = System.in
                   ^

Return code: 1
